rpc/transaction.go

```diff
@@ -55,7 +55,10 @@ func (s *PublicTransactionService) GetAccountNonce(
 	blockNum := blockNumber.EthBlockNumber()
 
 	// Response output is the same for all versions
-	addr := internal_common.ParseAddr(address)
+	addr, err := internal_common.ParseAddr(address)
+	if err != nil {
+		return 0, err
```

Comment:
does it make sense to return 0 here?

Reply:
it's returning the error. So this value is not meaningful and shouldn't be used anyway. 0 as a default value is fine.